generated-sources.json

```diff
@@ -0,0 +1,1109 @@
+[
```

Comment:
Extraneous file.

Reply:
Hm, it was in the npm folder, I thought it was also in the yarn folder as well. Should I remove it?

Reply:
Well this is in the root folder?

Reply:
Uh, this is embarrassing. It *was* on the root folder, and overwriting it with the actual yarn generate sources shows there is no difference, which I guess is a good regression test.